src/main.cpp

```diff
@@ -118,6 +123,12 @@ void setupSigtermHandler()
 
 Q_DECL_EXPORT int main(int argc, char **argv)
 {
+    if (argc > 1) {
+        if (strcmp(argv[1],"-d") == 0) {
+            toggleDebug = true;
+        }
+    }
+
     QCoreApplication app(argc, argv);
```

Comment:
Should QCoreApplication be allowed to handle its options first, if there are any?